frontend/src/pages/sites/sites.tsx

```diff
@@ -129,7 +139,7 @@ const Sites: React.FC = () => {
                   </span>
                 </Link>
               </div>
-              <h2>Landing</h2>
+              <h2>{projectName}</h2>
```

Comment:
```suggestion
              <h2>{project.name}</h2>
```
project should be always be here. use data status to show a loader

Reply:
should I avoid optional chaining operator here?

Reply:
yes. because you should be 100% sure that there is  project
you can use cast type `(project as Type).name`